src/test/java/uk/gov/pay/adminusers/resources/InviteResourceOtpIT.java

```diff
@@ -183,6 +185,58 @@ void validateOtp_shouldFailWith401_whenInvalidOtp() throws Exception {
                 .statusCode(UNAUTHORIZED.getStatusCode());
     }
 
+    @Test
+    void sendOtp_shouldSendNotification_whenInviteHasTelephoneNumber() {
+        code = InviteDbFixture.inviteDbFixture(databaseHelper)
+                .withTelephoneNumber("01234567890")
+                .withOtpKey(OTP_KEY)
+                .insertInviteToAddUserToService();
+
+        givenSetup()
+                .when()
+                .contentType(JSON)
+                .post(format(INVITES_SEND_OTP_RESOURCE_URL, code))
+                .then()
+                .statusCode(NO_CONTENT.getStatusCode());
+        
+        // TODO - validate that notification is sent?
```

Comment:
Yes it would be good if the integration test verified that a mocked endpoint on Notify is called. However, we currently start a DropWizard app to mock out Notify rather than using WireMock so this would be difficult. Perhaps we can leave this as is for now, and then in another PR look into how we could use WireMock to mock out Notify so we can do the verification.

Reply:
Agreed.  Should I remove the TODO now, or is it okay to leave it as we know we'll be following this up immediately?

Reply:
Raised as https://payments-platform.atlassian.net/browse/PP-10368; please have a look and see if I should change anything.